### ISSUE TITLE
Typescript: Syntax errors when formatted with without semi colon

### ISSUE THREAD

**Issue by tusharmath** [prettier/prettier#1930]
**Version:** `v1.4.1`

**Input**
```ts
type A = {disabled?: boolean, error?: string}
```

**Expected**
```ts
type A = {disabled?: boolean, error?: string}
```

**Actual**
```ts
type A = {disabled?: boolean error?: string} // missing comma
```

[Sample]

[Sample]: https://prettier.github.io/prettier/#%7B%22content%22%3A%22type%20A%20%3D%20%7Bdisabled%3F%3A%20boolean%2C%20error%3F%3A%20string%7D%22%2C%22options%22%3A%7B%22printWidth%22%3A80%2C%22tabWidth%22%3A2%2C%22singleQuote%22%3Afalse%2C%22trailingComma%22%3A%22none%22%2C%22bracketSpacing%22%3Afalse%2C%22jsxBracketSameLine%22%3Afalse%2C%22parser%22%3A%22typescript%22%2C%22semi%22%3Afalse%2C%22useTabs%22%3Afalse%2C%22doc%22%3Afalse%7D%7D

**Comment by tdeschryver** [comment]
a quick fix would be to format your type as
```
type A = {
  disabled?: boolean, 
  error?: string
}
```
This will not fix the semi colon, but won't give you syntax errors.

**Comment by azz** [comment]
Two possible fixes.

1. always print newline:
    ```ts
    type A = {
      disabled?: boolean
      error?: string
    }
    ```

2. respect source and use semi-colons.
    ```ts
    type A = {disabled?: boolean; error?: string}
    ```

**Comment by vjeux** [comment]
I just published 1.4.2 with this fix.

**Comment by tusharmath** [comment]
@vjeux Thanks!
